src/components/Form/fields.jsx

```diff
@@ -1,27 +1,22 @@
 import React from 'react'
-import Input from 'cozy-ui/transpiled/react/Input'
-import Textarea from 'cozy-ui/transpiled/react/Textarea'
+import TextField from 'cozy-ui/transpiled/react/MuiCozyTheme/TextField'
 
-function adaptField(WrappedComponent) {
-  class AdaptField extends React.Component {
-    render() {
-      const {
-        input,
-        meta: { valid },
-        ...rest
-      } = this.props
-      return <WrappedComponent {...input} {...rest} data-valid={valid} />
-    }
-  }
-  AdaptField.displayName = `adaptField(${getDisplayName(WrappedComponent)})`
-  return AdaptField
-}
-
-function getDisplayName(WrappedComponent) {
-  return WrappedComponent.displayName || WrappedComponent.name || 'Component'
-}
-
-const AdaptedInput = adaptField(Input)
-const AdaptedTextarea = adaptField(Textarea)
-
-export { AdaptedInput, AdaptedTextarea }
+export const DumbField = ({
```

Comment:
Since we only export one component, we can make it the default export imo. And we can also change the file name.

Reply:
Effectivement. Je rajoute aussi les proptypes